snapcraft.yaml

```diff
@@ -304,30 +290,37 @@ parts:
     prime:
       - -etc/fonts
       - -var
-      - -include
-      - -share/man
-      - -share/doc
+      - -usr/include
+      - -usr/lib/*/pkgconfig
+      - -usr/lib/*/libqpdf.a
+      - -usr/lib/cmake
       - -share/lintian
       - -usr/share/fonts
       - -usr/share/man
       - -usr/share/doc
       - -usr/share/doc-base
       - -usr/share/lintian
-      - -usr/lib/libqpdf.a
 
   ghostscript:
     # use GitHub mirror of GhostPDL here, as original GIT, self-hosted by
     # Artifex is not supported by ubuntu/desktop-snaps
     #source: https://git.ghostscript.com/ghostpdl.git
     source: https://github.com/ArtifexSoftware/ghostpdl.git
     source-type: git
-    source-tag: 'ghostpdl-10.05.1rc1_test002'
+    source-tag: 'ghostpdl-10.05.1'
     source-depth: 1
```

Comment:
In a separate PR we need to correct the Snap update automation here, so that for Ghostscript the beta, RC, and/or test releases get skipped.

Reply:
Yes. I'd rather implement the automation from snapcrafters here.